src/options/containers/settings/index.jsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react'
 import { Link } from 'react-router'
-import styles from './styles'
 
 class SettingsContainer extends React.Component {
     render() {
         return (
             <div>
-                <h1 style={styles.title}>Settings</h1>
+                <h1 className="routeTitle">Settings</h1>
```

Comment:
Instead of making `routeTitle` a global class, would it not be cleaner to `import { routeTitle } from '../../base.css'`?